<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>I2C_Ctrl</class>
 <widget class="QDialog" name="I2C_Ctrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>I2C控制</string>
  </property>
  <widget class="QPushButton" name="pushButton_read">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>读取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_write">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>175</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>写入</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_address">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>36</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_subaddress">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>36</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>38</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>设备地址： 0x</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>38</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>设备子地址：0x</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_Read">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>131</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_Write">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>204</y>
     <width>271</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>76</y>
     <width>43</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>读模式：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_havestop">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>63</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>有停止位</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_nostop">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>83</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>无停止位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>9</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请用十六进制填写地址和数据：</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>24</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>102</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>读取字节：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_readbits">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>101</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
